pkgs/applications/printing/pappl/default.nix

```diff
@@ -13,23 +13,15 @@
 
 stdenv.mkDerivation rec {
   pname = "pappl";
-  version = "1.1.0";
+  version = "1.3.1";
 
   src = fetchFromGitHub {
     owner = "michaelrsweet";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-FsmR0fFb9bU9G3oUyJU1eDLcoZ6OQ2//TINlPrW6lU0=";
+    hash = "sha256-96CC6QbtsSvjqQp7VbibvjoqR2HY0pjG4QGZDtw6ok8=";
   };
 
-  patches = [
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
resolved in rebased commits